.github/fixtures/test-github-integration/cliff.toml

```diff
@@ -31,21 +30,22 @@ body = """
   {% raw %}\n{% endraw %}
 {% endif %}
 """
-# remove the leading and trailing whitespace from the template
-trim = true
-# changelog footer
-footer = """
+# Whether to remove leading and trailing whitespaces from all lines of the changelog's body.
+trim_body_whitespace = true
+# A Tera template to be rendered as the changelog's footer (see https://keats.github.io/tera/docs/#introduction).
+footer_template = """
 <!-- generated by git-cliff -->
 """
 
-[git]
-# parse the commits based on https://www.conventionalcommits.org
-conventional_commits = false
-# filter out the commits that are not conventional
-filter_unconventional = true
-# process each line of a commit as an individual commit
-split_commits = false
-# regex for preprocessing the commit messages
-commit_preprocessors = [{ pattern = '\((\w+\s)?#([0-9]+)\)', replace = "" }]
-# protect breaking changes from being skipped due to matching a skipping commit_parser
-protect_breaking_commits = false
+[commit]
+# Whether to parse commits according to the conventional commits specification.
+# Sets the commits' `group` (= `type`), `scope`, `message` (= `description`), `body`, `breaking`, `breaking_description` and `footers`.
+parse_conventional_commits = false
+# Whether to exclude commits that do not match the conventional commits specification from the changelog.
+exclude_unconventional_commits = true
```

Comment:
I like `exclude` more than `filter`. It's more clear 👍 
Since we are already in the `[commit]` section, should we drop `_commits`? Same for `parse_conventional_commits`